dao/seeds/source_types.yml

```
@@ -66,6 +66,24 @@ amazon:
           pattern: "^arnaws:.*"
         - type: min-length
           threshold: 10
+    - type: provisioning-service-arn
+      name: Provisioning service's ARN
```

Just not to get this skipped, it's also here:

```suggestion
      name: Provisioning's ARN
```